docker-compose.yaml

```diff
@@ -35,7 +35,7 @@ services:
 
   # Want a two node Redpanda cluster? Uncomment this block :)
   # redpanda-2:
-  #   image: docker.redpanda.com/vectorized/redpanda:v22.3.5
+  #   image: docker.redpanda.com/vectorized/redpanda:v23.1.8
```

Comment:
Good catch, and thanks for bumping the version. For consistency with the redpanda-1 image, can we point to the `redpandadata` version of the image? `docker.redpanda.com/redpandadata/redpanda:v23.1.8`

Reply:
Thanks :)